docs/docs/configuration/secrets.md

```diff
@@ -29,13 +23,47 @@ To do so, export a `username=secret` environment variable where the value of tha
 
 To support this, the `secret` must not be set in the connector's configuration file **and** there musn't be any **Azure key vault** configured in the main configuration file.
 
-# Put the secrets in Azure Key Vaults
+
+# Put the secrets in CyberArk
+**CyberArk** (http://www.cyberark.com) Application Access Manager for DevOps provides a secrets management solution tailored specifically to the unique requirements of native-cloud and DevOps environments. The solution manages secrets and credentials used by non-human identities including DevOps and PaaS tools, and containers. **Prancer** validation framework supports secrets to be read from the **CyberArk**. Here is the steps:
+
+1. DevOps Engineer starts the **Prancer** validation framework.
+2. **Prancer** validation framework is integrated with CyberArk to retrieve secrets
+    a. CyberArk Agent has to be installed and configured on the respective servers (containers) and the firewall ports has to be opened 
```

Comment:
Have to be opened

Reply:
This part is weird, I dont feel it should be written like that. I think that part should be part of the configuration of the service not part of the explanation how prancer resolves the secrets

Reply:
let's have it here until we find a better place to put it in